glean-core/rlb/src/test.rs

```diff
@@ -354,6 +354,128 @@ fn test_dirty_flag_is_reset_to_false() {
     todo!()
 }
 
+#[test]
+fn setting_debug_view_tag_before_initialization_should_not_crash() {
+    let _lock = lock_test();
+
+    destroy_glean(true);
+    assert!(!was_initialize_called());
+
+    // Define a fake uploader that reports back the submission headers
```

Comment:
Wonder if we should turn this pattern into a helper function eventually (not necessary to do now)

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1679796